src/platform_impl/windows/event_loop.rs

```diff
@@ -265,10 +265,37 @@ impl<T> EventLoopWindowTarget<T> {
     }
 }
 
+/// Returns the id of the main thread.
+///
+/// Windows has no real API to check if the current executing thread is the "main thread" unlike macOS.
+/// Unix-like OSes have some APIs or semantics that exist regarding whether the main thread is in use or
+/// if the main thread id equals the pid like on Linux.
+///
+/// Windows will let us look up the current thread's id but this leaves us with the question of what
+/// is the main thread's id?
+/// We would somehow need to get the main thread's id before a developer could spin off any other
+/// threads inside of the main entrypoint in order to emulate the capabilities of other platforms.
+///
+/// We can get the id of the main thread by using CRT initialization.
+/// CRT initialization can be used to setup global state within a program by calling a list of
+/// function pointers which should assign a value to a static field.
+/// To have the field's value set to the main thread id, we need to place our function pointer inside
+/// of the section `.CRT$XCU` so it is called before the main entrypoint.
```

Comment:
```suggestion
/// To have get a hold of the main thread id, we need to place our function pointer inside
/// of the `.CRT$XCU` section so it is called before the main entrypoint.
```

I'm not sure what you mean by "To have the field's value set".